packages/hoppscotch-data/package.json

```diff
@@ -33,13 +35,15 @@
   "homepage": "https://github.com/hoppscotch/hoppscotch#readme",
   "devDependencies": {
     "@types/lodash": "^4.14.181",
-    "typescript": "^4.6.3",
+    "typescript": "^5.2.2",
     "vite": "^3.2.3"
   },
   "dependencies": {
     "fp-ts": "^2.11.10",
     "io-ts": "^2.2.16",
     "lodash": "^4.17.21",
-    "parser-ts": "^0.6.16"
+    "parser-ts": "^0.6.16",
+    "verzod": "^0.1.1",
+    "zod": "^3.22.2"
```

Comment:
Not connected to this PR, but i see that the verzod package has zod as a dependency. but probably we can move it to a peer dependency ?

Reply:
Will look into that. But you are right. Zod can be a peer dependency.